Directory.Build.props

```
@@ -86,7 +86,7 @@
     <TargetingPackInstallerBaseName>aspnetcore-targeting-pack</TargetingPackInstallerBaseName>
 
     <!-- Used to only produce targeting pack installers/packages once per major.minor. -->
-    <IsTargetingPackBuilding Condition="'$(IsServicingBuild)' == 'true' AND '$(DotNetBuildFromSource)' != 'true'">false</IsTargetingPackBuilding>
+    <IsTargetingPackBuilding Condition="'$(AspNetCorePatchVersion)' == '0' AND '$(DotNetBuildFromSource)' != 'true'">false</IsTargetingPackBuilding>
```

@wtgodbe when you're fixing up the use of `$(IsTargetingPackBuilding) in the Microsoft.AspNetCore.App.Ref project, please review this condition carefully. I suspect the second part was wrong before and now both parts are off e.g.
```suggestion
    <IsTargetingPackBuilding Condition="'$(AspNetCorePatchVersion)' != '0' OR '$(DotNetBuildFromSource)' == 'true'">false</IsTargetingPackBuilding>
```
Note `$(IsTargetingPackBuilding)` is always either empty or `false`